docs/source/markdown/podman-buildfarm.1.md

```diff
@@ -0,0 +1,34 @@
+% podman-buildfarm 1
+
+## NAME
+podman\-buildfarm - Farm out builds to machines running podman for different architectures
+
+## SYNOPSIS
+**podman buildfarm** *subcommand*
+
+## DESCRIPTION
+Farm out builds to machines running podman for different architectures.
+
+Manage farms by creating, updating, and removing them.
+
+## COMMANDS
+
+| Command  | Man Page                                                                      | Description                                                |
+| -------- | ----------------------------------------------------------------------------- | ---------------------------------------------------------- |
+| create      | [podman-buildfarm\-create(1)](podman-buildfarm-create.1.md)         | Create a new farm                  |
+| list     | [podman-buildfarm\-list(1)](podman-buildfarm-list.1.md)       | List the existing farms             |
+| remove   | [podman-buildfarm\-remove(1)](podman-buildfarm-remove.1.md)   | Delete one or more farms                                   |
+| update   | [podman-buildfarm\-update(1)](podman-buildfarm-update.1.md)   | Update an existing farm                  |
+
+## EXAMPLE
```

Comment:
Nit: this is not an example of `podman buildfarm`, but of `podman buildfarm list`
Remove in future PR.

Reply:
fixed